src/app/static/src/js/components/workflowWizard/step.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div v-if="active">
+        <div>
+            <slot></slot>
+        </div>
+        <div v-if="number !== 1" class="pt-4">
+            <button type="button"
+                    class="btn btn-sm btn-secondary disabled"
+                    name="cancel" @click="jump('cancel')">Cancel
+            </button>
+            <button v-if="number > 2" type="button" class="btn btn-sm btn-primary"
```

Comment:
It would be nice to keep all the knowledge about what actions should be enabled for what steps encapsulated in the runWorkflowRun component, it's already doing a lot of that management. Maybe nextVisible, runVisible etc could be passed into the step as props, maybe originally defined in the parent components array of steps. (Although these are somewhat dynamic e.g. Run workflow's back button should not be visible when we chose to re-run a workflow.)